flink-runtime/src/main/java/org/apache/flink/runtime/taskmanager/RuntimeEnvironment.java

```diff
@@ -92,6 +94,10 @@
 
     private final Task containingTask;
 
+    private MailboxExecutor mainMailboxExecutor;
```

Comment:
`@Nullable` and ditto for `asyncOperationsThreadPool` an `MockEnvironment`

Reply:
I mark the `mainMailboxExecutor ` and `asyncOperationsThreadPool` @Nullable in RuntimeEnvironment.

In `MockEnvironment`, both mailboxExecutor and asycnThreadPool are initialized in the constructor, and I marked the parameter of the set method as not nullable. So I did not set `mainMailboxExecutor ` and `asyncOperationsThreadPool` as @Nullable in MockEnvironment, because no way `mainMailboxExecutor ` and `asyncOperationsThreadPool` can be null in `MockEnvironment`.